README.md

```diff
@@ -9,47 +9,66 @@
       src="./assets/Safe_Paths_Logo.png"
     />
   </a>
-
-  <p>
-    Applying the technology and philosophy of Private Kit to COVID-19
-  </p>
   
-  <b>**https://covidsafepaths.org | https://safepaths.mit.edu**</b>
+  <b>**https://covidsafepaths.org**</b>
 </div>
 
 <hr />
 
+# Project Overview
+
 Help us stop COVID-19.
 
-We’re building the next generation of secure location logging to preserve privacy and _#flattenthecurve_
+COVID Safe Paths is a mobile app for digital contract tracing (DCT) sponsored by Path Check a nonprofit and developed by a growing global community of engineers, designers, and contributors. Safe Paths is based on research originally conducted at the MIT Media Lab. 
 
-Location logs provide time-stamped records of where users have been, allowing them to share information with health officials accurately and quickly. This helps support contact tracing efforts to slow the spread of the virus.
+## Privacy Preserving 
 
-What’s truly special about Safe Paths, though, is its privacy protection.
+What’s truly special about Safe Paths is our strong commitment to preserving the privacy of individual users. We're building an application that can help contain outbreaks of COVID-19 without forcing users to sacrafice their personal privacy. For example, if a user chooses to use Safe Paths to maintain a record of their locations use a time stamped GPS log, which stores 14 days of data in 5 minute increments. The location log is stored on the users phone. The logged data only leave the device if the user chooses to send the information to an authorized public health authority (PHA) as part of the contact tracing prcoess.
```

Comment:
The logged data only leave[s] the device if the user chooses to send the information to an authorized public health authority (PHA) as part of the contact tracing [process].